products/developments/sql/_bis.sql

```diff
@@ -2,7 +2,7 @@
 DESCRIPTION:
 	Initial field mapping and prelimilary data cleaning for BIS job applications data
 
-INPUTS: 
```

Comment:
Sort of funny that sqlfluff also hunts down trailing whitespace in comments. Fine by me!